client/src/site-search/search-results.tsx

```diff
@@ -286,93 +288,93 @@ function Results({
   const [searchParams] = useSearchParams();
 
   return (
-    <div>
-      <div className="search-results">
-        <p>
-          Found <ShowTotal total={metadata.total} /> in {metadata.took_ms}{" "}
-          milliseconds.
-        </p>
-
-        {!!suggestions.length && (
-          <div className="suggestions">
-            <p>Did you mean...</p>
-            <ul>
-              {suggestions.map((suggestion) => {
-                return (
-                  <li key={suggestion.text}>
-                    <Link
-                      to={`?${appendURL(searchParams, {
-                        q: suggestion.text,
-                        page: undefined,
-                      })}`}
-                    >
-                      {suggestion.text}
-                    </Link>{" "}
-                    <ShowTotal total={suggestion.total} />
-                  </li>
-                );
-              })}
-            </ul>
-          </div>
-        )}
+    <div className="search-results">
+      <p>
+        Found <ShowTotal total={metadata.total} /> in {metadata.took_ms}{" "}
+        milliseconds.
+      </p>
 
+      {!!suggestions.length && (
+        <div className="search-suggestions">
+          <p>Did you mean...</p>
+          <ul className="search-suggestion-list">
```

Comment:
```suggestion
          <ul>
```
it's not used by the SCSS and you'd be able to use `.search-suggestions ul` if we have to.

Reply:
Adding a class to a `ul` or `ol` for that matter causes the reset to kick in and remove some default styling that we would otherwise have to manually reset.